docs/index.rst

```diff
@@ -19,11 +19,13 @@ Catkit2
    :maxdepth: 1
    :caption: Built-In Services
 
-   service/aimtti_plp
+   services/aimtti_plp
```

Comment:
I think the name of the service is aimtti_plp_device

Reply:
You're right! However, the file that contains the service documentation is just called aimtti_plp (see [here](https://github.com/spacetelescope/catkit2/tree/develop/docs/services) for all existing files containing service docs), and this is what we want to link to from the `index.rst`.